projects/power_distribution/inc/pd_fan_ctrl_defs.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include "adt7476a_fan_controller.h"
+
+// I2C info - TODO: Figure out what's needed
```

Comment:
Nit: the TODO should have a ticket number, I'm not actually sure how this passed linting without it